installer/roles/kubernetes/templates/launch_awx.yml.j2

```diff
@@ -17,6 +17,9 @@ data:
     ANSIBLE_REMOTE_TEMP=/tmp ANSIBLE_LOCAL_TEMP=/tmp ansible -i "127.0.0.1," -c local -v -m wait_for -a "host=$DATABASE_HOST port=$DATABASE_PORT" all
     ANSIBLE_REMOTE_TEMP=/tmp ANSIBLE_LOCAL_TEMP=/tmp ansible -i "127.0.0.1," -c local -v -m wait_for -a "path=/var/run/redis/redis.sock" all
 
+{% if ca_trust_dir is defined %}
+    update-ca-trust extract
```

Comment:
Given that the launch script runs as user `awx` (not root), will this command actually work?

Reply:
From what I can see in my environment, **awx** user belong to '**root**' group.
```sh-4.4$ id
uid=1000(awx) gid=0(root) groups=0(root)
```
That may be enough to run the command?

Furthermore the command is already used in the '**local_docker**' role to do the same job.